.github/workflows/ci.yml

```diff
@@ -301,7 +305,7 @@ jobs:
         image_name: ${{ fromJson(needs.build-setup.outputs.image_names) }}
 
     env:
-      PLATFORMS: "${{ join(fromJSON(needs.build-setup.outputs.platforms), ',') }}"
+      PLATFORMS: "${{ join(fromJson(needs.build-setup.outputs.platforms), ',') }}"
```

Comment:
According to the docs it needs to uppercase I think? https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#fromjson

But seems to work 🤷